src/content/en/showcase/2017/bear71.md

```diff
@@ -0,0 +1,50 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-02-02T18:00:00.000Z #}
+{# wf_updated_on: 2017-02-02T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/bear71/featured.png #}
+{# wf_featured_snippet: Bear 71 is an interactive documentary launched by the National Film Board of Canada in 2012 to much critical acclaim. When it was built, Flash was in its golden years but with the deprecation of browser support for Flash, we were eager to explore WebVR, both as a way to preserve the project’s legacy and a chance to innovate how the story is experienced. #}
+{# wf_tags: webvr,casestudy #}
+
+# Bear 71 and WebVR {: .page-title }
+
+<img src="/web/showcase/2017/images/bear71/featured.png" class="attempt-right">
+
+### TL;DR {: .hide-from-toc }
```

Comment:
We should add some text and a link to the Bear71 VR experience at the top, in fact we should do that for all the case studies. This is the URL for Bear71 (note, it's not being released there until next week but this is the correct  URL) - https://nfb.ca/bear71vr/